### ISSUE TITLE
Transfer Learning not working on 1.0.0-beta3

### ISSUE THREAD

**Issue by klevis** [deeplearning4j/deeplearning4j#6713]
Hi,
OS : Ubuntu 18

When I run below code on  1.0.0-beta3 the cost function printed on console it is like the model was never trained before. While if I run on  1.0.0-beta2 everything looks fine. Additionally if we are going to train directly the ComputationGraph(vgg16) everything works fine. Only when ComputationGraph is wrapped with TransferLearning object it doesn't work. 
 Let me know if any other detail is needed, maybe I can provide the zip model as well...
```
ComputationGraph vgg16 = ModelSerializer.restoreComputationGraph(
                new File(MODEL_SAVE_PATH + "611_epoch_data_e1024_b64_850.zip"));
       IUpdater iUpdaterWithDefaultConfig = Updater.ADAM.getIUpdaterWithDefaultConfig();
        FineTuneConfiguration fineTuneConf = new FineTuneConfiguration.Builder()
                .seed(1234)
                .optimizationAlgo(OptimizationAlgorithm.STOCHASTIC_GRADIENT_DESCENT)
                .activation(Activation.RELU)
                .updater(iUpdaterWithDefaultConfig)
                .cudnnAlgoMode(ConvolutionLayer.AlgoMode.USER_SPECIFIED)
                .miniBatch(true)
                .inferenceWorkspaceMode(WorkspaceMode.ENABLED)
                .trainingWorkspaceMode(WorkspaceMode.ENABLED)
                .pretrain(true)
                .backprop(true)
                .build();
 ComputationGraph cifar10 = new TransferLearning.GraphBuilder(vgg16)
                .setWorkspaceMode(WorkspaceMode.ENABLED)
                .fineTuneConfiguration(fineTuneConf)
                .build();
DataSetIterator dataSetIterator = ImageUtils.createDataSetIterator(rootDir, NUM_POSSIBLE_LABELS, BATCH_SIZE);
        DataSetIterator testIterator = ImageUtils.createDataSetIterator(rootDir, NUM_POSSIBLE_LABELS, BATCH_SIZE);

cifar10.setListeners(new ScoreIterationListener(1));
        int iEpoch = 0;
        int iIteration = 0;
        while (iEpoch < EPOCH_TRAINING) {
            while (dataSetIterator.hasNext()) {
                DataSet trainMiniBatchData = null;
                try {
                    trainMiniBatchData = dataSetIterator.next();
                } catch (Exception e) {
                    e.printStackTrace();
                }
                cifar10.fit(trainMiniBatchData);
                iIteration++;
            }
            iEpoch++;

                Evaluation eval = cifar10.evaluate(testIterator);
                System.out.println(eval.stats());
                testIterator.reset();
               dataSetIterator.reset();

        }
```

**Comment by AlexDBlack** [comment]
So, I've run some basic checks with vgg16 pretrained model. The parameters are definitely transferred between networks correctly and everything looks fine to me.
My next guess is you're seeing a difference between versions: see the first entry for the 1.0.0-beta3 deeplearning4j "bug fixes and optimizations" section:
https://deeplearning4j.org/release-notes.html#deeplearning4j-bug-fixes-and-optimizations
The net effect of that bug is that the optimal learning rate for some updaters with scaling (Adam, RMSProp etc) needs to be reduced vs. what was used in 1.0.0-beta2 and before.
It's unfortunate that nobody picked that up sooner, but behaviour should be more in line with outher libraries when using these updaters/optimizers.

I'd suggest trying with a smaller learning rate. If that doesn't work, I'll need more details to look into this further.

**Comment by AlexDBlack** [comment]
Closing this due to lack of follow-up.
Happy to take another look if required, but I'm not sure there's a problem here.
